packages/@dcl/react-ecs/src/components/Button/index.tsx

```diff
@@ -53,10 +53,11 @@ export function Button(props: UiButtonProps) {
     height: 36,
     ...uiTransform
   })
+
   return (
     <entity
-      onMouseDown={onMouseDown}
-      onMouseUp={onMouseUp}
+      onMouseDown={props.disabled ? undefined : onMouseDown}
```

Comment:
should we added some logic to the primary/secondary buttons so when the Button is disabled we change the background color ?
This way the user will know when the button is disabled or not

Reply:
Totally, I wasn't sure why there were the primary or secondary variants and now it makes sense XD, I'll update it, so if the new 'disabled' property is being used, based on that I'll force either the 'primary' or 'secondary' variant

Reply:
Done! found a way of affecting the alpha of the text and background colors regardless of their configuration, check it out